stdlib/file/src/main/ballerina/src/file/file.bal

```diff
@@ -151,11 +178,14 @@ function externReadDir(handle path, int maxDepth) returns FileInfo[]|Error = @ja
 
 # Copy file/directory in old path to new path.
 # If new path already exists, this replaces the file.
+# ```ballerina
+# file:Error? results = file:copy("/A/B/C", "/A/B/D", true);
+# ```
 #
-# + sourcePath - String value of old file path.
-# + destinationPath - String value of new file path.
-# + replaceExisting - Flag to allow replace if file already exists in destination path.
-# + return - Returns an `Error` if failed to rename.
+# + sourcePath - String value of old file path
+# + destinationPath - String value of new file path
+# + replaceExisting - Flag to allow replace if file already exists in destination path
+# + return - An `Error` if failed to rename
```

Comment:
```suggestion
# + return - An `Error` if failed to rename
```
Do we need to specify the error?

Reply:
The encompassing Error object is named as `Error`, I've added the module name to be more clear. Added in dba6e94